hide-items.css

```
@@ -55,7 +55,13 @@ div.bar-icon { /* 隐藏聊天页面上侧的语音通话，视频通话，屏
     display: none; 
 }
 
-html > body.q-theme-tokens-light > div.lite-tools-vue-component.vue-component > div.container > div.nav-bar-container.lite-tools-vue-component.vue-component > div > div.nav-item > div > div.nav-item-container.lite-tools-vue-component.vue-component:last-child {
+div#app > div.container:nth-of-type(2) > div.nav-bar-container.vibrancy.lite-tools-vue-component.vue-component:nth-of-type(1) > div:nth-of-type(2) > div.wifiphoto-btn {
```

必须安装了对应插件才会有这两个 class，建议移除

```suggestion
div#app > div.container:nth-of-type(2) > div.nav-bar-container.vibrancy:nth-of-type(1) > div:nth-of-type(2) > div.wifiphoto-btn {
```